ethpm/validation.py

```diff
@@ -1,14 +1,25 @@
 import re
 from urllib import parse
 
-from eth_utils import is_checksum_address
+from eth_utils import is_canonical_address, is_checksum_address
 
 from ethpm.constants import PACKAGE_NAME_REGEX, REGISTRY_URI_SCHEME
 from ethpm.exceptions import ValidationError
 from ethpm.utils.ipfs import is_ipfs_uri
 from ethpm.utils.registry import is_ens_domain
 
 
+def validate_address(address: bytes) -> None:
```

Comment:
Normally validation functions should accept `Any` since you could in theory pass anything in.